src/components/common/Actions.vue

```diff
@@ -134,7 +134,7 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
   private async onClickFilePay (): Promise<void> {
     this.setIsFilingPaying(true)
     const filing = await this.buildFiling()
-    await this.saveFiling(filing, false)
+    await this.completeFiling(filing)
```

Comment:
If we do error handling in this component, do we need to splits up completeFiling into 2 separate calls -- the first to save and the second to file?

Reply:
I'll look into it!